m2m-transfers/V3/m2m-transfers-json-contract-v3.md

```diff
@@ -376,42 +376,42 @@ EDITTRAN - Edit existing transaction (expire existing transfer & create a new tr
-|            | Config file error: [configuration file name] read error - [system generated letter file read error message] ||
-|            | Config file error: 3-Error Reading Letterfile [configuration file name]: [system generated letter file read error message] ||
-|            | Config file error: Invalid Parameter in CFG file ||
-| 502        | Invalid Source Account: Pref Access type 3 not found ||
-|            | Invalid Source Account: Acct Warning 1234 ||
-|            | Invalid Source Account: No eligible transfer from shares/loans ||
-|            | Invalid Source Account: Acct Type 1234 ||
-| 503        | Invalid Recipient Account ||
-| 504        | Insufficient Information: Cannot calc member limits. ||
-| 505        | Invalid Input ||
-| 506        | Error Processing Recipient Record | Error creating/deleting recipient |
-| 507        | Error Processing Transfer Record | Error creating/updating/deleting transfer |
-|            | Error Processing Transfer Record: Target s/l xfer Loc 1234567 not found ||
-|            | Error Processing Transfer Record: Target ID not found or invalid ||
-| 508        | Undefined Error ||
-| 509        | Member verification failed | Member unverified |
-|            | Member verification failed: Member account not found ||
-|            | Member verification failed: Account Closed - 99/99/99 ||
-|            | Member verification failed: Name verification failed ||
-|            | Member verification failed: Must be a different member account number ||
-|            | Member verification failed: S/L closed or charged-off - 99/99/99 ||
-|            | Member verification failed: S/L ID not found ||
-|            | Member verification failed: No valid share or loan found ||
-|            | Member verification failed: S/L missing service code ||
-|            | Member verification failed: Invalid S/L type ||
-|            | Member verification failed: Invalid share code ||
-|            | Member verification failed: Loan has $0.00 payoff ||
-|            | Member verification failed: Invalid IRS code ||
-| 510        | Account ID incorrect ||
-| 511        | Request exceeds limits ||
+*See Modifier section for additional details
+
+| Error Code | Logging Error Message                                                                 | Modifier                                                                             | Additional Notes As Needed                    |
+| ---------- | ------------------------------------------------------------------------------------- |------------------------------------------------------------------------------------- |-----------------------------------------------|
+| 500        | Program running in memo mode |||
+| 501        | Config file error | : [configuration file name] open error - [system generated letter file read error message] | This set of errors represents issues with the configuration letter file setup |
```

Comment:
Same as the v1. Consider combining col 2 and 4 into one statement for col 2 and eliminate col 4 altogether.

Reply:
I've removed the following additional notes from M2M V1 

'This set of errors represents issues with the configuration letter file setup'
and
'This set of errors represents issues with the source account'